financialaid/models.py

```diff
@@ -166,3 +166,11 @@ class FinancialAidAudit(TimestampedModel):
     financial_aid = models.ForeignKey(FinancialAid, null=True, on_delete=models.SET_NULL)
     data_before = JSONField(blank=True, null=False)
     data_after = JSONField(blank=True, null=False)
+
+
+class CurrencyExchangeRate(TimestampedModel):
+    """
+    Table of currency exchange rates
```

Comment:
Can you update this to mention that the `exchange_rate` field is the amount of foreign currency per single USD, just to be explicit?